python-backend/constants.py

```diff
@@ -8,3 +8,5 @@
 DEBUG_LINES = 100
 # evalf allows for verbose output via param universally set to this:
 VERBOSE_EVAL = False
+# chunk size to send to front end via web socket when computing coordinate pairs
+BATCH_SIZE = 250
```

Comment:
just curiosity, why not a power of 2?

Reply:
Normally I would for non-user-facing numbers. I chose a random big-but-not-too-big number and never thought much about it once everything worked. I think I was looking at it from the human side since a person will be looking at the charts and humans like tens.